docs/graph-algorithms/breadth-first-search.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Breadth First Search
+slug: /graph-algorithms/breadth-first-search
+---
+
+## Introduction
+Breadth First Search Algorithm (BFS) is a Popular algorith to find shortest path in unwighted graph
```

Comment:
This does not explain anything. Spell mistakes here and there. I would suggest a rewrite of this portion.